assess/config/themes_mapping.py

```diff
@@ -122,3 +122,38 @@ def ordered_themes(fund_round_short_name):
         ]
     if fund_round_short_name == "CTDFCR1":
         return ["project_name", "organisation_name"]
+    if fund_round_short_name == "HSRAVR":
+        return [
+            "what-would-you-like-to-name-your-application?",
```

Comment:
Is it worth pulling out the common themes into a shared variable? Might make it easier to see at a glance what is the same between both rounds. 

Happy either way.

Reply:
What is the themes mapping for? I thought the order of the themes is decided by the assessment config.

Reply:
Good shout @ksp37-dluhc , will add the changes 👍

Reply:
@albertkol this one is used to specify the order of information shown in the "View entire application" feature in assessment.

https://github.com/communitiesuk/funding-service-pre-award-frontend/blob/e457b3fc15a5a117962cf3e1d12b23b70d3c73ad/assess/themes/deprecated_theme_mapper.py#L271

<img width="353" alt="Screenshot 2024-12-24 at 12 34 28" src="https://github.com/user-attachments/assets/bd8fe9cd-0968-4194-b0bb-805ee2607a1d" />